src/application/Application.js

```diff
@@ -1,8 +1,76 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import { Col, Progress, Button, Form, FormGroup, Input, Label } from 'reactstrap';
 
 function Application({ id, firstName, lastName }) {
-  return <div>Application {id}: {firstName} {lastName} </div>;
+  return (
+    <div>
+      <Col md="9">
+        CV
+      </Col>
+      <Col md="3">
```

Comment:
If we keep the CV on the left and answers to questions on the right, we could have the review options as a ribbon at the bottom, which would give us more space. I think we should really try to get all the information for a single candidate on a single screen.